db/re/item_combo_db.txt

```diff
@@ -218,6 +218,7 @@
 2527:5467:13030,{ bonus2 bAddRace,RC_Dragon,25; }
 2527:13001,{ bonus2 bAddRace,RC_Dragon,5; }
 2527:13030,{ bonus2 bAddRace,RC_Dragon,5; }
+2584:5579,{ .@r = getequiprefinerycnt(EQI_HEAD_TOP); .@bonus = max(getskilllv("TF_DOUBLE"),5); if(.@r>=5) { skill "TF_DOUBLE",1; bonus bDoubleRate,25; } else if(.@r>=7) { skill "TF_DOUBLE",.@bonus; bonus bDoubleRate,.@bonus*5; } }
```

Comment:
I believe the bonus is using the refine lvl of `Wanderer_Outer` (EQI_GARMENT)
https://www.divine-pride.net/database/item/2584
Additionnally `.@bonus = max(getskilllv("TF_DOUBLE"),5);` is only used when the refine is +7 or above, move that in the >=7 condition

Reply:
I didn't understand.
Its already for >=7 condition.

```if(.@r>=5) { skill "TF_DOUBLE",1; bonus bDoubleRate,25; }```
```else if(.@r>=7) { skill "TF_DOUBLE",.@bonus; bonus bDoubleRate,.@bonus*5; } }```

Or its wrong?